roles/hyperthreadingdisabled/tasks/main.yml

```diff
@@ -1,13 +1,10 @@
 # Download latest version from DOMjudge repo
 # Alternative is to put the correct file in /tmp and set internet to false
-- name: Download turboboost script
+- name: Download turboboost script  # noqa: risky-file-permissions
   delegate_to: localhost
   ansible.builtin.get_url:
     url: https://raw.githubusercontent.com/DOMjudge/domjudge-scripts/main/provision-contest/disable-turboboost_ht
     dest: /tmp/turboboost_ht
-    mode: 0777
```

Comment:
This one can stay, and than the problem is fixed, the {user,group} is not checked in the risky-filepermissions